os-server/src/main/java/app/openschool/discussion/DiscussionAnswerMentorController.java

```diff
@@ -0,0 +1,54 @@
+package app.openschool.discussion;
+
+import app.openschool.common.response.ResponseMessage;
+import app.openschool.discussion.dto.DiscussionAnswerRequestDto;
+import app.openschool.discussion.dto.DiscussionAnswerResponseDto;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import java.security.Principal;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1/discussionAnswerMentor")
```

Comment:
/api/v1/courses/{courseId}/mentor-questions for mentor questions
/api/v1/courses/{courseId}/peers-questions for peers questions

accordingly, the answers api will be 
/api/v1/courses/{courseId}/mentor-questions/{questionId}/answer for mentor answer
/api/v1/courses/{courseId}/peers-questions/{questionId}/answer for peers answer